README.md

```diff
@@ -132,24 +170,24 @@ lti13.enableTokenController=false
  - The remaining new env vars `lti13.enableDynamicRegistration`, `lti13.enableRoleTesting`, `lti13.enableTokenController` are for test endpoints. `lti13.enableDynamicRegistration` corresponds to the `RegistrationController` which has not been tested. `lti13.enableRoleTesting` corresponds to the `TestController` which I suspect will be deleted as role authorization is not needed. `lti13.enableTokenController` corresponds to `TokenController` which I suspect will be deleted if it remains unused.
 
-Setup SQS for AGS Grade Pass Back
----------------------------------
+## Setup SQS for AGS Grade Pass Back
+
 1. The authentication to AWS uses Spring's [default authentication chain](https://cloud.spring.io/spring-cloud-static/spring-cloud-aws/1.2.3.RELEASE/multi/multi__basic_setup.html#_sdk_credentials_configuration).
 2. The URI for a SQS queue should be filled in for either `lti13.grade-passback-queue` or as `LTI13_GRADE_PASSBACK_QUEUE` environment variable.
-3. The region of the SQS queue should be filled in for either `cloud.aws.region.static` in the application.properties file or `AWS_REGION` as an environment variable.
+3. The region of the SQS queue should be filled in for either `cloud.aws.region.static` in the `.properties` file or `AWS_REGION` as an environment variable.
+
+## Turning off AWS Integration for Local Development
 
-Turning off AWS Integration for Local Development
--------------------------------------------------
 Prior to running `mvn clean install spring-boot:run`, do the following:
 1. `export spring_profiles_active=no-aws`
-2. Ensure that the following property is present and uncommented in the application-local.properties file (or application.properties, whichever is being used locally):
+2. Ensure that the following property is present and uncommented in the `.properties` file:
 `spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration,org.springframework.cloud.aws.autoconfigure.context.ContextCredentialsAutoConfiguration,org.springframework.cloud.aws.autoconfigure.context.ContextInstanceDataAutoConfiguration,org.springframework.cloud.aws.autoconfigure.context.ContextRegionProviderAutoConfiguration,org.springframework.cloud.aws.autoconfigure.context.ContextResourceLoaderAutoConfiguration,org.springframework.cloud.aws.autoconfigure.context.ContextStackAutoConfiguration`
 
-Turning on AWS Integration for Local Development
-------------------------------------------------
+## Turning on AWS Integration for Local Development
+
 Prior to running `mvn clean install spring-boot:run`, do the following:
 1. `unset spring_profiles_active`
-2. Ensure that the following properties are set in the application-local.properties file (or application.properties, whichever is being used locally):
+2. Ensure that the following properties are set in the `.properties` file:
```

Comment:
Are the following properties all still required? I don't see them in the base `application.properties`. 

I was able to comment out the following in my local properties:
```
lti13.grade-passback-queue=<queue name>
cloud.aws.credentials.access-key=<access key>
cloud.aws.credentials.secret-key=<secret key>
cloud.aws.end-point.uri=<sqs uri>
```

But I am wondering if I was ok not needing them as I had the values stored in my environment variables?

We might also want to make a note to use `envchain `to store and use the environment variables instead of hardcoding them in, if there are two paths there?

Reply:
That's a good point. I added a set of prerequisite instructions to the "Turing on AWS Integration for Local Development" section that explains how to install and setup envchain. Then I removed `cloud.aws.credentials.*` and `cloud.aws.end-point.uri` from the list of properties that need to be set. I suspect that the region and grade-passback-queue properties could be set in either the envchain or the properties file; I don't think it matters which one so I left those in the properties list.

I also updated the "Turing off AWS Integration for Local Development" section now that I know how to set spring arguments in the run command, so it's no longer necessary to set an environment variable to turn off the AWS integration because that can be done in the same step as running the application.